DatadogWebViewTracking/Sources/WebViewTracking.swift

```diff
@@ -0,0 +1,57 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import DatadogInternal
+
+/// Defines methods to send WebView related information
+public protocol WebViewTracking {
+    /// Sends a bag of data
+    /// - Parameter body: The data to send, it must be parsable to `WebViewTrackingMessage`
+    func send(body: Any) throws
+
+    /// Sends a message
+    /// - Parameter message: The message to send
+    func send(message: WebViewTrackingMessage) throws
+}
+
+/// Datadog implementation of `WebViewTracking`
+/// - Note:
+/// Cross platform SDKs should instantiate this type with a `DatadogCoreProtocol` implementation
+/// and pass WebView related messages using the message bus of the core.
+public struct WebViewTrackingCore: WebViewTracking {
+    enum MessageKeys {
+        static let browserLog = "browser-log"
+        static let browserRUMEvent = "browser-rum-event"
+    }
+
+    private let core: DatadogCoreProtocol
```

Comment:
/blocking We should keep a `weak` ref to the core. Even if we won't have any retain cycle with webview tracking, it's best not to retain the core so we can control its deallocation from the future core registry.

Reply:
that makes sense, I made it weak.